Header header
int32 central_position
int32 backhoe_position
StateString state
Duty duty
State state_i

